robot/web/web_test.go

```diff
@@ -929,10 +929,6 @@ func TestRawClientOperation(t *testing.T) {
 		return []robot.Status{}, nil
 	}
 
-	iRobot.(*inject.Robot).StatusFunc = func(ctx context.Context, resourceNames []resource.Name) ([]robot.Status, error) {
```

Comment:
@cheukt pointed out that these lines were a noop and likely an accidental copy of the ones directly above.